admin/tests/unit/controllers/authenticated/certifications/certification/informations_test.js

```diff
@@ -425,6 +438,41 @@ module('Unit | Controller | authenticated/certifications/certification/informati
     });
   });
 
+  module('#onCancelCourseButtonClick', () => {
+    test('should enable cancel course confirm dialog', async (assert) => {
+      // when
+      await controller.onCancelCourseButtonClick();
+      // then
+      assert.equal(controller.confirmAction, 'onCancelCourseConfirmation');
+      assert.ok(controller.displayConfirm);
```

Comment:
On peut tester mieux que ça, ici en principe on sait exactement à quoi s'attendre du point de vue du contenu de ces variables. assert.ok ça teste simplement que c'est not null / undefined, c'est pas suffisant